pkgs/by-name/ma/materialgram/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  telegram-desktop,
+  lib,
+  xdg-utils,
+  stdenv,
+  fetchFromGitHub,
+}:
+telegram-desktop.overrideAttrs (
+  finalAttrs: previousAttrs: {
+    pname = "materialgram";
+    version = "5.1.7.1";
+
+    src = fetchFromGitHub {
+      owner = "kukuruzka165";
+      repo = "materialgram";
+      rev = "v${finalAttrs.version}";
+      fetchSubmodules = true;
+      hash = "sha256-OlXcDsjZ59/GjysfFAOo6dAoNnUY5hBD9NuNgTR4rXc=";
+    };
+
+    postFixup =
```

Comment:
We shall not duplicate this part in every tdesktop fork.... Lemme do a quick preparatory PR to clean things up.

Reply:
https://github.com/NixOS/nixpkgs/pull/324630